bobtemplates/plone/addon/buildout.cfg.bob

```diff
@@ -91,6 +91,12 @@ eggs = zest.releaser
 recipe = zc.recipe.egg
 eggs = i18ndude
 
+
+[translate]
```

Comment:
please pay a bit more attention on how you name things!
What does this part do?
Also this is the name which will be placed into the bin folder.
So we want something there, which makes sense.
And please generate a package and test if it works ;)